packages/babel/README.md

```diff
@@ -95,7 +95,7 @@ We recommend to follow those guidelines for each possible value:
 - `'runtime'` - you should use it especially when building libraries with rollup. It has to be used in combination with `@babel/plugin-transform-runtime` and you should also specify `@babel/runtime` as dependency of your package (don't forget to tell rollup to treat it as your external dependency when bundling for `cjs` & `es` formats).
 - `'bundled'` - you should use it if you want your resulting bundle to contain those helpers (at most one copy of each). Useful especially if you bundle an application code.
 - `'external'` - use it only if you know what you are doing. It will reference helpers on **global** `babelHelpers` object. Used in combination with `@babel/plugin-external-helpers`.
-- `'inline'` - this is not recommended. Helpers will be inserted in each file using them. This can cause serious code duplication (this is the default Babel behaviour).
+- `'inline'` - this is not recommended. Helpers will be inserted in each file using them. This can cause serious code duplication. **(for reference, this is the default behaviour of Babel, but it is overridden by the `babelHelpers` value of this plugin: to either your own explicit selection, otherwise the default value of `'bundled'` will be used)**
```

Comment:
```suggestion
- `'inline'` - this is not recommended. Helpers will be inserted in each file using them. This can cause serious code duplication. This is the default Babel behavior as Babel operates on isolated files, as Rollup is a bundler and is project-aware the default of this plugin is `"bundled"`.
```

How about this one?

Reply:
@Andarist made a further amendment to your suggestion: 14fe234